### PR TITLE
Drop type specs for L1Trigger, Muon and RecoB

### PR DESCRIPTION
**Pull request by NiharSaha**
#### PR description:
Drop type specs in DQMOffline/L1Trigger, DQMOffline/Muon, DQMOffline/RecoB python configuration files.

Central DQM migration campaign for drop type spces following
https://cms-sw.github.io/cms_coding_rules.html#6--packaging-rules-1
and update the safer syntax for existing parameters like: (1) drop type specifications where the original parameter exists. (2) using "clone" instead of "deepcopy" (3) move all parameters inside the clone

#### PR validation:
runTheMatrix.py -l limited -i all --ibeos

### CODE REVIEW

File: DQMOffline/L1Trigger/python/L1TTauDiff_cfi.py [hunk 1]
```diff
@@ -45,11 +45,11 @@
 l1tTauEmuDiff = l1tDiffHarvesting.clone(
     plotCfgs=cms.untracked.VPSet(
         cms.untracked.PSet(  # EMU comparison
```
Comment (jfernan2) [inline DQMOffline/L1Trigger/python/L1TTauDiff_cfi.py:47]: Shouldn't this be converted to a dict and drop type afterward?

**Comment by jfernan2** [comment]
@perrotta @qliphy  the bot reports conflicts in this branch, do you know how to check those conflicts? They seem to not be reported

**Comment by perrotta** [comment]
> @perrotta @qliphy the bot reports conflicts in this branch, do you know how to check those conflicts? They seem to not be reported

Those configs have been moved away by #36627
Please rebase and apply the updates to the files in the new location

File: DQMOffline/L1Trigger/python/L1TTauDiff_cfi.py (hunk 1 above)
Comment (NiharSaha) [reply DQMOffline/L1Trigger/python/L1TTauDiff_cfi.py:47]: Yeah, I have tried that, but it was showing error. That's why I left them unchanged. May I know if Pset->dist, then VPSet->??

**Pull request closed without merging**